core-web/libs/portlets/dot-experiments/data-access/src/lib/services/dot-experiments.service.ts

```diff
@@ -51,11 +51,11 @@ export class DotExperimentsService {
     /**
      * Get an array of experiments of a pageId filter by status
      * @param {string} pageId
-     * @param {DotExperimentStatusList} status
+     * @param {DotExperimentStatus} status
      * @returns Observable<DotExperiment[]>
      * @memberof DotExperimentsService
      */
-    getByStatus(pageId: string, status: DotExperimentStatusList): Observable<DotExperiment[]> {
+    getByStatus(pageId: string, status: DotExperimentStatus): Observable<DotExperiment[]> {
         return this.http
```

Comment:
Check this because it shows an ts error:

```
Type 'Observable<DotExperiment[] | undefined>' is not assignable to type 'Observable<DotExperiment[]>'.
  Type 'DotExperiment[] | undefined' is not assignable to type 'DotExperiment[]'.
    Type 'undefined' is not assignable to type 'DotExperiment[]'.ts(2322)
```

Reply:
And there are others like this one.

Reply:
The problem comes from 
```
.pipe(pluck('entity'));
```
Because the interface 
```
export interface DotCMSResponse<T> {
    contentlets?: T;
    entity?: T;
    tempFiles?: T; // /api/v1/temp don't have entity
    errors: string[];
    i18nMessagesMap: { [key: string]: string };
    messages: string[];
    permissions: string[];
}
```
`entity` is optional `?`, and the expected response from the http is `: Observable<DotExperiment>` but with the `?` can be undefined too I will add undefined. 

I created an extended interface from `DotCMSResponse` and make `entity` required only form Experiment Service.